src/app/components/ModeRanking.js

```diff
@@ -0,0 +1,78 @@
+export const ModeRanking = (topScores) => {
+  const modeRankingObj = document.createElement('div');
+  modeRankingObj.classList.add('modeRanking');
+
+  const headerContainer = document.createElement('div');
+  headerContainer.classList.add('modeRanking__headerContainer');
+
+  const rankingIcon = document.createElement('span');
+  rankingIcon.classList.add('modeRanking__icon');
+  headerContainer.appendChild(rankingIcon);
+
+  const rankingHeader = document.createElement('h2');
+  rankingHeader.classList.add('modeRanking__header');
+  rankingHeader.innerText = 'Mode Ranking';
+  headerContainer.appendChild(rankingHeader);
+
+  modeRankingObj.appendChild(headerContainer);
+
+  const rankingContainer = document.createElement('div');
+  rankingContainer.classList.add('modeRanking__rankingContainer');
+
+  const placesList = ['1st', '2nd', '3rd'];
+
+  const titleRow = document.createElement('header');
+  titleRow.classList.add(
+    'modeRanking__rankingRow',
+    'modeRanking__rankingRow--title',
+  );
+
+  const headerElementTitles = ['Place', 'Player', 'Answered'];
+
+  for (let i = 0; i < headerElementTitles.length; i++) {
+    rankingContainer.appendChild(titleRow);
+    const headerElement = document.createElement('div');
+    headerElement.classList.add(
+      'modeRanking__headerItem',
+      `modeRanking__headerItem--${headerElementTitles[i].toLowerCase()}`,
+    );
+    headerElement.textContent = headerElementTitles[i];
+    titleRow.appendChild(headerElement);
+  }
+
+  if (typeof topScores !== 'undefined' && topScores.length > 0) {
```

Comment:
Tutaj wystarczy: 
`if (topScores !== undefined && topScores.length > 0)`
Albo w ogóle :
`if (!topScores && topScores.length > 0)`

Reply:
A najlepiej to nazwać też co to dla nasz znaczy. Czyli:

```
const rankingIsEmpty = !topScores && topScores.length > 0
if(rankingIsEmpty){
/...cos tam tutaj..//
}
```

a jak się pomęczysz i wydorebnisz komponenty o których piszę niżej to może się uda coś w stylu nawet:
```
rankingContainer = rankingIsEmpty ? EmptyRanking() : RankingWithScores()
```
oczywiscie jakies tam argumenty do tych funkcji.

Reply:
hmmm fajne rozwiązanie! :-)